schema/project.js

```diff
@@ -305,7 +305,12 @@ class Project extends BaseModel {
       .leftJoinRelation('licenceHolder')
       .withGraphFetched('[licenceHolder, additionalEstablishments(constrainAAParams), establishment(constrainEstParams)]')
       .modifiers({
-        constrainAAParams: builder => builder.select('id', 'name', 'projectEstablishments.status'),
+        constrainAAParams: builder => {
+          builder.select('id', 'name', 'projectEstablishments.status');
+          if (status !== 'inactive') {
```

Comment:
Is this really confusingly describing draft project statuses and not thing described in the UI as `Inactive`?

Reply:
Yeah, this is value passed when the draft tab is selected - the "Inactive" tab sends `inactive-statuses`

Reply:
Could we maybe add a unit test?

Reply:
Done.